lib/ansible/cli/__init__.py

```diff
@@ -228,14 +228,26 @@ def normalize_become_options(self):
         self.options.become_ask_pass = self.options.become_ask_pass or self.options.ask_sudo_pass or self.options.ask_su_pass or C.DEFAULT_BECOME_ASK_PASS
         self.options.become_user     = self.options.become_user or self.options.sudo_user or self.options.su_user or C.DEFAULT_BECOME_USER
 
+        def _dep(which):
+            display.deprecated('%s command line options have been deprecated, use the equivalent become one instead' % which, '2.6')
```

Comment:
Probably some potential for improved grammar here.

> use the equivalent become one instead

Maybe something more like:

> The %s command line option has been deprecated in favor of the "become" command line arguments